pkgs/by-name/sc/screenconfig/package.nix

```
@@ -0,0 +1,40 @@
+{
+  lib,
+  python3,
+  fetchFromGitHub,
+  xrandr,
+  srandrd,
+  feh,
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "screenconfig";
+  version = "0.1.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "jceb";
+    repo = "screenconfig";
+    tag = "v${version}";
+    hash = "sha256-X1Mz8UbOOW/4LM9IZoG/kbwv2G0EppTsacKapQMChkc=";
+  };
+  build-system = [ python3.pkgs.setuptools ];
+  dependencies = with python3.pkgs; [
+    toml
+  ];
+
+  buildInputs = [
```

```suggestion
  propagatedBuildInputs = [
```
Otherwise they will not be available at runtime.

---

Done.